### ISSUE TITLE
WAI-ARIA & Panels & Panopoly

### ISSUE THREAD

**Issue by mgifford** [wet-boew/wet-boew-drupal#88]
Posted a note here about panels & aria support.  Certainly live regions should be added and be able to be added to describe content areas.
http://drupal.org/node/1681940

I have to look more at the code and learn how we can add more semantic meaning to this rich new interface.  Without that it could be quite an accessibility challenge.

**Comment by sylus** [comment]
So would the plan be to make the modal dialog and subsequent "live regions" when customizing panels layout have appropriate aria support?

I'd love to work on this but would need some guidance :)

**Comment by mgifford** [comment]
I'm trying to drum up some ideas about this, but there are lots of places we know there are going to be problems.

Drag/Drop functionality is an easy one to highlight, but ultimately it is going to take someone sitting down with a screenreader or without a mouse and trying to do something with Panels.

Has anyone tried to administer the distro without a mouse?  That's an easy way to find issues.  I don't have time just yet to do that.

**Comment by StephenOTT** [comment]
What is the level of accessibility compliance for the admin side of the distro that we are trying to meet?

**Comment by mgifford** [comment]
With Drupal 7 the goal has been WCAG 2.0 AA. There has not been enough work do on ATAG 2.0, but the goal discussed for Drupal 7 was also AA.  

With so many sites being interactive, the distinction (especially when designing for a general use case) between admin functionality & user functionality is blurred.  Interactive forms, customizable dashboards, comments are all pieces which are now more expected, even on government sites.  

The distro's mandate I expect is focused on the required WCAG 2.0 AA for the public pieces of the website.

**Comment by sylus** [comment]
This can be further discussed in the Drupal.org community but is most certainly out of scope for this repo.
